### ISSUE TITLE
Support WHITELIST_PATH

### ISSUE THREAD

**Issue by ericfranz** [OSC/ood-myjobs#275]
Add a config method called `whitelist_paths` and `whitelist_paths?`.

```ruby
Configuration.whitelist_paths
# => [] if WHITELIST_PATH env var not set
# => ['/fs/project', '/fs/scratch', '/users'] if WHITELIST_PATH env var is "/fs/project:/fs/scratch:/users"
Configuration.whitelist_paths?
# => returns Configuration.whitelist_paths.any?
```

The ability to create a new job from specified path: https://ondemand.osc.edu/pun/sys/myjobs/new_from_path

Add a model validation on `workflow[staging_template_dir]` i.e. the Source Path field in the web form if the path that is specified is not a path under one of the whitelist paths. The validation should only work if whitelist_paths?

**Comment by ericfranz** [comment]
You can of course test this by setting WHITELIST_PATH in the .env file when running the application.

**Comment by MorganRodgers** [comment]
Fixed in #277.
